projects/public-search/src/app/routes/documents-route.service.spec.ts

```diff
@@ -16,6 +16,7 @@
  */
 
 import { TestBed } from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
```

Comment:
same for this file, all changes could be removed

Reply:
I believe these changes are needed because of the changes made in file documents-route.service.ts